test/full-mentions.js

```diff
@@ -48,30 +48,25 @@ test('getMessagesByMention', (t) => {
       db.publish(mentionMsg, (err) => {
         t.error(err, 'no err')
 
-        db.onDrain('fullMentions', () => {
-          const status = db.getStatus()
-          t.equal(status.indexes['fullMentions'], 780, 'index in sync')
```

Comment:
This isn't using `getStatus` anymore, because that would have to happen after the `onDrain('fullMentions'`, but the onDrain is done implicitly inside the custom operator `mentions()`, so I didn't know where to put this one...

Reply:
should be fine, we really just care about the results